python/Ganga/Core/GangaRepository/GangaRepositoryXML.py

```diff
@@ -239,11 +239,12 @@ def shutdown(self):
         from Ganga.Utility.logging import getLogger
         logger = getLogger()
         logger.debug("Shutting Down GangaRepositoryLocal: %s" % self.registry.name)
-        for k in self._fully_loaded:
-            try:
-                self.index_write(k, shutdown=True)
-            except Exception as err:
-                logger.error("Warning: problem writing index object with id %s" % k)
+        for k in self.objects:
+            if self.objects[k]._inMemory:
```

Comment:
This is a matter of personal taste, but I find:
```
for k, obj in self.objects.iteritems()
   if obj._inMemory():
```
easier to parse.
I understand that my nitpicking can get annoying, please let me know if I should direct my energy elsewhere.